scripts/spec-exec-disasm-audit.sh

```
@@ -0,0 +1,88 @@
+
+audit_riscv64() {
+  local tgt=riscv64gc-unknown-none-elf
+  note "building arch-riscv64 $tgt --release"
+  if ! cargo build -p smallaios-arch-riscv64 --target "$tgt" --release 2>/dev/null; then
+    note "riscv64 build unavailable — skip"
+    return 0
+  fi
+  note "riscv64: scaffolding phase — fence.i placeholder only, full CFI pending Zicfilp ratification"
+}
+
+for a in $ARCHES; do
```

Shellcheck (0.11.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)